docs/README.md

```diff
@@ -0,0 +1,30 @@
+# Firebase Android SDK
+
+The Firebase SDK for Android is the official way to add Firebase to your
+Android app. To get started, visit the [setup instructions][android-setup].
+
+## Open Source
+
+The following Firebase SDKs for Android are developed in this
+repository:
+
+  * `firebase-common`
+  * `firebase-database`
+  * `firebase-firestore`
+  * `firebase-functions`
+  * `firebase-inappmessaging-display`
+  * `firebase-storage`
+
+For more information on building the SDKs from source or contributing,
+visit the [main README][main-readme].
+
+## Kotlin Extensions
+
+The following Firebase SDKs for Android have Kotlin extension libraries
+that allow you to write more idiomatic Kotlin code when using Firebase
+in your app:
+
+  * [`firebase-firestore`](./ktx/firestore.md)
```

Comment:
Would it make sense to include `firebase-common` here? I know we don't have a place for it on firebase devsite, but imo it could be beneficial here, just to show how to get a hold of default/non-default apps as well as FirebaseOptions. wdyt?

```kotlin
val defaultApp = Firebase.app
val defaultOptions = Firebase.options

val anotherApp = Firebase.initialize(context, Firebase.options, "myApp")
```

Reply:
Yeah that was the plan, to make a separate page for each `ktx` library.  Are those the only two right now?

Reply:
Yep, those are the only two